// webpack-partial/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.loader = exports.merge = exports.output = exports.plugin = exports.tap = exports.inject = exports.partial = exports.root = exports.alias = undefined;

var _partial = require('./partial');

var _partial2 = _interopRequireDefault(_partial);

var _inject = require('./inject');

var _inject2 = _interopRequireDefault(_inject);

var _tap = require('./tap');

var _tap2 = _interopRequireDefault(_tap);

var _plugin = require('./plugin');

var _plugin2 = _interopRequireDefault(_plugin);

var _output = require('./output');

var _output2 = _interopRequireDefault(_output);

var _merge = require('./merge');

var _merge2 = _interopRequireDefault(_merge);

var _loader = require('./loader');

var _loader2 = _interopRequireDefault(_loader);

var _alias = require('./alias');

var _alias2 = _interopRequireDefault(_alias);

var _root = require('./root');

var _root2 = _interopRequireDefault(_root);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.alias = _alias2.default;
exports.root = _root2.default;
exports.partial = _partial2.default;
exports.inject = _inject2.default;
exports.tap = _tap2.default;
exports.plugin = _plugin2.default;
exports.output = _output2.default;
exports.merge = _merge2.default;
exports.loader = _loader2.default;
exports.default = _partial2.default;
